src/TriangleNodesProvisioner.cpp

```diff
@@ -576,26 +637,24 @@ namespace blitzdg {
         index_hashmap& bcMap = *BCmap;
 
         // create boundary face nodes global numbering.
-        index_matrix_type boundaryNodesMat(NumFacePoints, NumFaces*NumElements);
+        index_matrix_type boundaryNodesMat(NumFacePoints, NumFaces*NumElements, ColumnMajorOrder());
```

Comment:
Any reason `boundaryNodesMat` is column-major here?

Reply:
I had to do this to make the test pass after putting in the other changes. If we switch to row-major, and the solver still works, then I'm happy so I'll change the list of nodes in the test and everything should be just dandy.